edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/handlers/PolicyUpdateHandler.cs

```diff
@@ -78,7 +78,6 @@ static PolicyUpdate GetEmptyPolicy()
         {
             return new PolicyUpdate(@"
             {
-                'schemaVersion': '2020-10-30',
```

Comment:
I understand this is no longer needed, but why is it no longer needed?

Reply:
I talked to Varun about that. Since it won't be exposed to the customers as I thought in the beginning, there is no need to have that right now. We will have only "high level" twin schema version, and will use that for versioning policy changes (if any).